### ISSUE TITLE
How to install it

### ISSUE THREAD

**Issue by Kevinrob** [GabLeRoux/ansi-colors-chrome-extension#2]
How can I install it?
I know that this is a WIP, but stripping is a great feature 😄

**Comment by heavybeard** [comment]
I need it!!! :)

**Comment by GabLeRoux** [comment]
Cool, this plugin is getting some attention 😃 

I created [a release ⬇️](https://github.com/GabLeRoux/ansi-colors-chrome-extension/releases) from what's in the repo so it's easier to install. I added instructions in the Readme and improved overall documentation. 👍 

I don't think it's ready for the chrome web store yet, but I won't stop you from playing with the extension ;)

I wrote a small blog post about ansi colors here if interested:
https://gableroux.com/tips/2016/02/18/keep-saltstack-colored-output/

There's so much left to do, I'm still stuck with #1 and haven't found a solution yet, but if someone has a tip, I'd be happy to continue working on this. 😄
